Model/Sales/TrackTraceHolder.php

```diff
@@ -332,14 +330,34 @@ private function getWeightTypeOfOption(?string $weight): int
         return (int) $weight ?: 1000;
     }
 
+    /**
+     * Get country of origin from product settings or, if they are not found, from the MyParcel settings.
+     *
+     * @param $product_id
+     * @return string
+     */
+    public function getCountryOfOrigin($product_id): string
```

Comment:
Kijk of je van product_id een `int` kan maken. Hierdoor hoef je hem op regel 341 niet te casten.